src/sidebar/helpers/query-parser.ts

```diff
@@ -18,20 +23,41 @@ const filterFields = [
   'user',
 ];
 
+/**
+ * Names of fields that can be used in `{field}:{term}` queries with
+ * {@link parseHypothesisSearchQuery}.
+ */
+export type SearchField = 'group' | 'quote' | 'tag' | 'text' | 'uri' | 'user';
+
+const searchFields: SearchField[] = [
+  'group',
+  'quote',
+  'tag',
+  'text',
+  'uri',
+  'user',
+];
```

Comment:
This is a subset of the fields listed at https://h.readthedocs.io/en/latest/api-reference/#tag/annotations/paths/~1search/get.